package.json

```diff
@@ -17,34 +17,43 @@
     "script-shell": "bash"
   },
   "scripts": {
+    "explode": "node exploder.js",
     "format": "eslint ./../../ --ext .js --ext .html -c .eslintrc.json",
-    "util": "npm run install:sass && npm run install:copy && npm run install:css:minify && npm run install:fonts && npm run install:bamboo && npm run install:copy:github && npm run update:readme && npm run format",
-    "install:utils": "npm run install:bamboo && npm run install:copy:install && npm run install:copy:codeowner && npm run install:copy:github",
     "install:bamboo": "node ./bamboo/bamboo-spec.js",
-    "install:sass": "sass ../../src/style.scss ../../src/style.css --load-path ../../node_modules/@govflanders",
-    "install:copy": "npm run install:copy:install && npm run install:copy:test && npm run install:copy:release && npm run install:copy:codeowner && npm run install:copy:github && npm run install:copy:wct && npm run install:copy:mocharc && npm run install:copy:npmignore && npm run install:copy:eslint",
-    "install:copy:install": "npm i copyfiles@2.1.0 --save",
-    "install:copy:eslint": "cp .eslintrc.json ../../",
-    "install:copy:wct": "copyfiles *.conf.json ../../",
-    "install:copy:test": "copyfiles -f test/docker-compose.yml ../../test",
-    "install:copy:release": "copyfiles -f docker-compose-release.yml ../../",
+    "install:copy": "npm run install:copy:install && npm run install:copy:test && npm run install:copy:release && npm run install:copy:codeowner && npm run install:copy:github && npm run install:copy:wct && npm run install:copy:npmignore && npm run install:copy:eslint",
     "install:copy:codeowner": "copyfiles CODEOWNERS ../../",
+    "install:copy:eslint": "cp .eslintrc.json ../../",
     "install:copy:github": "cp -R .github ../../",
-    "install:copy:mocharc": "cp .mocharc.js ../../",
+    "install:copy:install": "npm i copyfiles@2.1.0 --save",
```

Comment:
Als een dependency configureren is niet voldoende?

Reply:
Geen idee, in ieder geval niet in de scope van dit ticket.